docs/new_experiments.md

```diff
@@ -0,0 +1,21 @@
+## Creating a new experiment
+
+- Create a new repository under the [faasm org](
+  https://github.com/faasm) called `experiment-<name>`.
+- Create a Dockerfile
+- Create benchmark scripts to run both the Faasm and native versions
+
+The directory structure should look like: 
+
+```bash
+|-experiment-base/
+|-experiment-newname/
+|---README.md
+|---Dockerfile
+|---bin/
+|------build_native.sh
+|------build_faasm.sh
+|---run/
+|------faasm.sh
+|------native.sh
```

Comment:
Using the same names for scripts across repos makes a lot more sense to me, then you really can just copy-paste. Having the scripts named after each experiment seems a bit awkward